Remapping bg-fill-magic-secondary colors and adding a new tone to Badge (#11303)
Shopify/polaris: .changeset/purple-mice-build.md

---
'@shopify/polaris': minor
'@shopify/polaris-tokens': minor
---

> Remapping bg-fill-magic-secondary colors and adding a Magic tone to Badge

Reviewer: ```suggestion
- Remapped the `bg-fill-magic-secondary` token colors
- Added a Magic `tone` to `Badge`
```
